bfdd/control.c

```diff
@@ -774,13 +774,13 @@ static void _control_notify(struct bfd_control_socket *bcs,
 	control_queue_enqueue(bcs, bcm);
 }
 
-int control_notify(struct bfd_session *bs)
+int control_notify(struct bfd_session *bs, uint8_t notify_state)
```

Comment:
Please, don't change this function signature. All calls of this function you are already using what is inside `struct bfd_session`.